### PR TITLE
Update qubesformation.py

### PR DESCRIPTION
**Pull request by ProfessorManhattan**
Hey, I thought this was a good idea since sometimes previous templates need to be created before another template based on that template is created. This is needed when you have a template based off another template but please correct me if I'm wrong.

### CODE REVIEW

**Comment by ProfessorManhattan** [comment]
I think I might be wrong on this... What's the difference between `template:` and `template_vm:`?

**Comment by ProfessorManhattan** [comment]
This might be necessary for the volume property? If we want VMs to inherit the volume size for root/private

File: action_plugins/qubesformation.py [hunk 1]
```diff
@@ -17,6 +17,13 @@ def generate_datastructure(vms, task_vars):
     dc = collections.OrderedDict
     d = dc()
     for n, data in vms.items():
+        # This block will skip any VMs that are not in the 'vms' variable
+        # This allows you to deploy in multiple stages which is useful in cases
+        # where you want to create a template after another template is already provisioned.
```
Comment (Rudd-O) [inline action_plugins/qubesformation.py:22]: That should not be necessary, the dependencies should already take care of deploying things in the right order.

Comment (Rudd-O) [reply action_plugins/qubesformation.py:20]: I see what you did here.  Merged.

**Review by Rudd-O — APPROVED** [review]

**Pull request merged**